### ISSUE TITLE
Can we uninstall this package and turn on csrutil with debug back after the Apple Intelligence has been installed on Macs?

### ISSUE THREAD

**Issue by TomHsiung** [CatMe0w/zouxian#9]
Hi,

Thanks for this great package. But we wonder: Can we uninstall this package and turn on csrutil with debug back after the Apple Intelligence has been installed on Macs? Many thanks.

**Comment by TomHsiung** [comment]
Looks like the zouxian service is essential for the presence of Apple Intelligence and Xcode Predictive Code Completion. I have tested by stopping the zouxian service before I rebooted my Mac. After the reboot, all Apple Intelligence and Xcode predictive Code Completion feature have gone.

Late I restarted the zouxian service the immediate they came back on my Mac. It is a little pity that we have to keep one part of the SIP function disabled.
